recipes/r-aroma.affymetrix/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
```

Comment:
@iwohlers can you remove all bat files from this PR. We do not support Windows in bioconda.